pages/forms/terms/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>We Life</title>
+  </head>
+  <body>
+    <header>
+      <title>Terms and Conditions</title>
+    </header>
+    <form action="../registration/index.html">
```

Comment:
There is no reason to have a form here. There is a single input, and we are including the whole page inside the form.